exercises/01-getting-started.md

```diff
@@ -1,63 +1,77 @@
 # Getting started
-In the following we describe how to get started with this project.
+
+In the following, we describe how to get started with this project.
 You will create a new **private** repository based on this repository and run the application locally to get familiar with it.
 
 > In the original session, the participants were handed out a participantId. If you go through the tasks on your own, make up your own id. We recommend to refrain from using personal data.
 
 ## Create a new repository based on this template
-Click the *Use this template* button on the top of the GitHub page.
+
+Click the _Use this template_ button on the top of the GitHub page.
```

Comment:
[req] Check the [style guide](https://github.com/SAP/cloud-sdk/blob/main/STYLEGUIDE.md) and ensure that the styles are corectly used. I think this is correct here, but please double check.

Reply:
I cahnge it along with [this style guide](https://github.com/SAP-samples/cloud-sdk-team-calendar/pull/27/files/2d490543e8066cfd789083cc6f7b09f1ca5a4b5d..61bb3a516622d6adf84b7fc2fff5a793c796cc5f)